setup.py

```diff
@@ -62,6 +62,7 @@
         ]
     },
     py_modules=['gswrap', 'gswrap_meta'],
-    include_package_data=True,
-    package_data={"gswrap": ["py.typed"]},
-    data_files=[('.', ['LICENSE', 'README.rst'])])
+    package_data={
+        "gswrap": ["py.typed"],
```

Comment:
Did this work, i.e. was `py.typed` included in the distribution?

Reply:
Yes, it did. I checked out some of your repos, e.g. pypackagery and a difference seemed to be this. After the change py.typed was included in the distribution although I'm not sure why my previous version didn't work.